QUARKUS-658.md

```diff
@@ -0,0 +1,49 @@
+# QUARKUS-658 - Reactive REST client - support HTTP/2
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-658
+
+PR: https://github.com/quarkusio/quarkus/pull/31192
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/13969
+
+Quarkus should be able to use http/2 on client side to be able to comunicate with other microservices.
+
+## Scope of the testing
+Scope of the test will cover basic usage of rest client inside quarkus. The complex scenarios like sending request to diferent API/microservice won't be covered.
+
+Scope in point will be:
+- Run Quarkus application and send requests using `http/2` to verify that server response with `http/2`. 
+- Run Quarkus application with enabled Application-Layer Protocol Negotiation (alpn) to verify that correct http version is used.
+- Check the usecases using HTTP and HTTPS.
+
+These points will be covered for single REST client and all REST client to verify all of these properties:
```

Comment:
```suggestion
These points will be covered for a single REST client and all REST clients to verify these properties:
```

Not sure what I should imagine under single REST client vs. all REST clients. Please elaborate.